src/Umbraco.Web.BackOffice/Controllers/UsersController.cs

```diff
@@ -179,8 +179,9 @@ public ActionResult<string[]> GetCurrentUserAvatarUrls()
         var fileName = file.FileName.Trim(new[] { '\"' }).TrimEnd();
         var safeFileName = fileName.ToSafeFileName(shortStringHelper);
         var ext = safeFileName.Substring(safeFileName.LastIndexOf('.') + 1).ToLower();
+        const string allowedAvatarFileTypes = "jpeg,jpg,gif,bmp,png,tiff,tif,webp";
 
-        if (contentSettings.DisallowedUploadFiles.Contains(ext) == false)
+        if (allowedAvatarFileTypes.Contains(ext) == true && contentSettings.DisallowedUploadFiles.Contains(ext) == false)
```

Comment:
We might want to limit the avatar image type to only supported image formats using [`_imageUrlGenerator.IsSupportedImageFormat(ext)`](https://github.com/umbraco/Umbraco-CMS/blob/e626fca2432582f052cb13654eedd9e60ef8723f/src/Umbraco.Core/Media/ImageUrlGeneratorExtensions.cs#L20) instead.

That will ensure we're generating image URLs that will be property processed here:
https://github.com/umbraco/Umbraco-CMS/blob/e626fca2432582f052cb13654eedd9e60ef8723f/src/Umbraco.Core/Models/UserExtensions.cs#L77-L101

Reply:
Given that there has not been bug reports for this before (when even more formats were allowed), it doesn't seem like an actual problem. If anyone wants to take it as an Up For Grabs then they're very welcome.